examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -21,25 +30,54 @@
 
 ###############################################################################
 # First we generate a 500x500 px image with varying frequency content:
-x = np.arange(500) / 500 - 0.5
-y = np.arange(500) / 500 - 0.5
+N = 450
+x = np.arange(N) / N - 0.5
+y = np.arange(N) / N - 0.5
+aa = np.ones((N, N))
+aa[::2, :] = -1
 
 X, Y = np.meshgrid(x, y)
 R = np.sqrt(X**2 + Y**2)
-f0 = 10
-k = 250
+f0 = 5
+k = 100
 a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
-
-
+# make the left hand side of this
+a[:int(N / 2), :][R[:int(N / 2), :] < 0.4] = -1
+a[:int(N / 2), :][R[:int(N / 2), :] < 0.3] = 1
+aa[:, int(N / 3):] = a[:, int(N / 3):]
+a = aa
 ###############################################################################
-# The following images are subsampled from 500 data pixels to 303 rendered
-# pixels. The Moire patterns in the 'nearest' interpolation are caused by the
-# high-frequency data being subsampled.  The 'antialiased' image
+# The following images are subsampled from 450 data pixels to either
+# 125 pixels or 250 pixels (depending on your display).
+# The Moire patterns in the 'nearest' interpolation are caused by the
+# high-frequency data being subsampled.  The 'antialiased' imaged
 # still has some Moire patterns as well, but they are greatly reduced.
-fig, axs = plt.subplots(1, 2, figsize=(7, 4), constrained_layout=True)
-for ax, interp in zip(axs, ['nearest', 'antialiased']):
-    ax.imshow(a, interpolation=interp, cmap='gray')
-    ax.set_title(f"interpolation='{interp}'")
+#
+# There are substantial differences between the 'data' interpolation and
+# the 'rgba' interpolation.  The alternating bands of red and blue on the
+# left third of the image are subsampled.  By interpolating in 'data' space
+# (the default) he antialiasing filter makes the stripes close to white,
+# because  the average of -1 and +1 is zero, and zero is white in this
```

Comment:
```suggestion
# (the default) the antialiasing filter makes the stripes close to white,
# because the average of -1 and +1 is zero, and zero is white in this
```